packages/core/src/exports.ts

```diff
@@ -318,3 +322,86 @@ export function getClient<C extends Client>(): C | undefined {
 export function getCurrentScope(): Scope {
   return getCurrentHub().getScope();
 }
+
+/**
+ * Start a session on the current isolation scope.
+ *
+ * @param context (optional) additional properties to be applied to the returned session object
+ *
+ * @returns the new active session
+ */
+export function startSession(context?: SessionContext): Session {
+  const client = getClient();
+  const isolationScope = getIsolationScope();
+
+  const { release, environment = DEFAULT_ENVIRONMENT } = (client && client.getOptions()) || {};
+
+  // Will fetch userAgent if called from browser sdk
+  const { userAgent } = GLOBAL_OBJ.navigator || {};
+
+  const session = makeSession({
+    release,
+    environment,
+    user: isolationScope.getUser(),
+    ...(userAgent && { userAgent }),
+    ...context,
+  });
+
+  // End existing session if there's one
+  const currentSession = isolationScope.getSession && isolationScope.getSession();
```

Comment:
l: IMHO no need to check for `getSession` existance, if you have code without this everything will fail here anyhow I guess xD